.github/workflows/release-please.yml

```diff
@@ -0,0 +1,18 @@
+on:
+    push:
+      branches:
+        - main
```

Comment:
Should this be run on every merge? I don't understand this action fully yet, but I'd imagine we would want it to be manually triggered, then it just grabs all the commits since the last release and creates a release with them. I could be mistaken.